src/controllers/user/userController.js

```diff
@@ -89,39 +89,38 @@ const userController = {
     const { email, password } = req.body;
     const [statusCode, result] = await userService.login(email, password);
 
-    return res
-      .status(statusCode)
-      .send(result);
+    return res.status(statusCode).send(result);
```

Comment:
```suggestion
    return res
      .status(statusCode)
      .send(result);
```
다른 파일들과 동일하게 작성해주세요!

Reply:
@chldppwls12 확인해보니 제 vs code 자체에서 정렬이 되어버리네요